apps/artwork/lib/inquire.coffee

```diff
@@ -11,6 +11,8 @@ module.exports = (id) ->
 
   Promise artwork.fetch()
     .then ->
+      artwork.set('is_in_auction', true) if sd.AUCTION?
```

Comment:
Doing this now instead. `sd.AUCTION` only gets set if `artwork.is_in_auction` is true. Since we are refetching the artwork to pass in to the inquiry_questionnaire, this is the most straightforward way of handling.